ffi/diplomat/Cargo.toml

```diff
@@ -66,28 +66,51 @@ cpp_default = ["provider_test", "logging", "simple_logger"]
 # meta feature for things we enable by default in wasm
 wasm_default = ["buffer_provider", "logging"]
 
+# Components
+default_components = ["icu_collator", "icu_datetime", "icu_decimal", "icu_displaynames", "icu_list",
+                      "icu_locid_transform", "icu_normalizer", "icu_plurals",
+                      "icu_properties", "icu_segmenter"]
+# icu_calendar = ["dep:icu_calendar"] # Calendars are not currently useful on their own
+icu_collator = ["dep:icu_collator"]
+# icu_collections = ["dep:icu_collections"] # Not useful on its own
```

Comment:
There's a single import from `icu_collections`: 
```rust
use icu_collections::codepointtrie::TrieValue;
```

We could reexport `TrieValue` from `icu_properties`. Do Rust clients need that for `icu_properties` to be functional?

Reply:
Not usually unless they're writing generic code.

Reply:
I thought we established that `icu_calendar` is useful on its own, and in Rust, `icu_collections` is definitely useful on its own.